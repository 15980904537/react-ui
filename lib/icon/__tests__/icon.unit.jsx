import renderer from 'react-test-renderer'
import React from 'react'
import {
    Icon
} from '../Icon'
describe("icon测试", () => {
            it('render successify', () => {
                    const json = renderer.create( < Icon name = { 'wechat' }
                        / > ).toJSON()
                        expect(json).toMatchSnapshot()
                    })
            });