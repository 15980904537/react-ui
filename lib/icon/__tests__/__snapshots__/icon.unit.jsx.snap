// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon测试 icon是否插入页面中 1`] = `
<span>
  <svg
    className="my-icon"
  >
    <use
      xlinkHref="#wechat"
    />
  </svg>
</span>
`;
